strawberry/config.py

```diff
@@ -0,0 +1,10 @@
+from enum import IntEnum
+
+class EnumGenerate(IntEnum):
+    MEMBER = 0
+    VALUE  = 1
+
+class StrawberryConfig:
+    auto_camelcase: bool = True
+    resolvers_strict_mode: bool = True
+    enum_values = EnumGenerate.MEMBER # or VALUE
```

Comment:
what do you think of only supporting auto_camel_case for now? So we can have this new configuration option that also provides a new future?

So I'd have this for now:

```
@dataclass
class StrawberryConfig:
    auto_camelcase: bool = True
```

Then we can expand in future?

Reply:
Yes, ok